### ISSUE TITLE
Better MC [FABRIC] - BMC2 Missing mods

### ISSUE THREAD

**Issue by JoniLieh** [LunaPixelStudios/Better-MC#1551]
### Describe the Bug

1. When installing the modpack, error message with sodium comes up - Sodium needs to be installed.
2. Quest Book has Quests for Eden Ring - Eden Ring is not included

### Reproduction Steps

1. Download and Install modpack
2. Run the instance

1. Open Quest Book for Eden Ring
2. Try to build the Eden Ring Portal

### Screenshots and Videos

![image](https://github.com/LunaPixelStudios/Better-MC/assets/57151672/8b8fabb4-78cf-4631-b652-23bbdabdbf3f)
```
Incompatible mods found!
net.fabricmc.loader.impl.FormattedException: Some of your mods are incompatible with the game or each other!
A potential solution has been determined, this may resolve your problem:
	 - Install sodium, version 0.5.3.
More details:
	 - Mod 'Indium' (indium) 1.0.27+mc1.20.1 requires version 0.5.3 of sodium, which is missing!
	 - Mod 'Iris' (iris) 1.6.11 requires version 0.5.3 of sodium, which is missing!
	 - Mod 'Reese's Sodium Options' (reeses-sodium-options) 1.6.5+mc1.20.1-build.95 requires any version of sodium, which is missing!
	 - Mod 'Sodium Extra' (sodium-extra) 0.5.1+mc1.20.1-build.112 requires version 0.5.1 or later of sodium, which is missing!
	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:51)
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:194)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:146)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:87)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:130)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
```

### Operating System

Windows 11

### Minecraft Version

1.20

### Modloader

Fabric

### Modpack Version

bmc v24

**Comment by chorbintime** [comment]
you didnt install the modpack correctly. please do a fresh install on ATLauncher

**Comment by JoniLieh** [comment]
Thanks for the info, regarding Sodium missing: The PrismLauncher didnt look for Modrinth download and went for curseforge instead, but Sodium moved to Modrinth...
And the Eden Ring Quest is in the book altough Eden Ring is not in the modpack since Eden Ring isnt released for 1.20. yet
